Construct Bin. Tree from inorder,preorder/Solution.java

```diff
@@ -0,0 +1,24 @@
+// Time Complexity: O(n)
```

Comment:
Kindy check the time complexity. Every time finding the root will take  O(n^2) .